test/specs/user.js

```diff
@@ -33,7 +33,7 @@ describe('/me', () => {
         .expect(200)
         .expect(response => {
           assert.equal(response.body.data.pil.reviewDate, '2025-01-01T12:00:00.000Z');
-          assert.equal(response.body.data.pil.reviewDue, false);
+          assert.equal(response.body.data.pil.reviewDue, true);
```

Comment:
I think the test behaviour should stay as is and instead update the PEL for the test user. As far as I can tell:
* The user for the api uses all the defaults https://github.com/UKHomeOffice/asl-public-api/blob/master/test/helpers/api.js#L26
* So the logged in id is `abc123` https://github.com/UKHomeOffice/asl-public-api/blob/master/test/helpers/with-user.js#L5
* This maps to Linford Christie: https://github.com/UKHomeOffice/asl-public-api/blob/master/test/data/default.js#L11
* So if https://github.com/UKHomeOffice/asl-public-api/blob/master/test/data/default.js#L730 is changed to be updated in 2024 that should restore the previous reviewDue = false behaviour